### PR TITLE
Hm dashboard tab added

### PR DESCRIPTION
**Pull request by Andrewsopoku**
**Issue**

- Link: https://github.com/NebraLtd/maker-starter-app/issues/152
- Summary: We should add dashboard as an extra tab. Even if, to start with, it's just a button that opens a browser window to Dashboard of similar it would be good to advertise it from the app

**How**

**Screenshots**

Simulator Screen Shot - iPhone 13 - 2022-06-08 at 16 42 03

**References**

**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: src/features/hmDashboard/HmDashboardScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          flex={1}
+          justifyContent="center"
+          backgroundColor="primaryBackground"
+          style={{
+            flexDirection: 'column',
+          }}
+        >
+          <WebView
+            source={{
+              uri: 'https://dashboard.nebra.com/',
```
Comment (marvinmarnold) [inline src/features/hmDashboard/HmDashboardScreen.tsx:32]: Will we be able to easily track clicks if we don't add a UTM? Is it possible to use Segment for tracking this?

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
The PR looks good but I want to make sure we have a way to easily track this. It's one of our explicit KPIs.

File: src/features/hmDashboard/HmDashboardScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import { AppScreens } from '../../utils/analytics/screens'
+
+const HmDashboardScreen = () => {
+  const { screen } = useAnalytics()
+
+  // Segment log screen visit
+  useEffect(() => {
+    screen(AppScreens.HMDASHBOARD)
+  }, [screen])
```
Comment (Andrewsopoku) [inline src/features/hmDashboard/HmDashboardScreen.tsx:17]: We track here.

Comment (marvinmarnold) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: Isn't this tracking tab load. Not link clicking?

Comment (Andrewsopoku) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: This is tracking screen visit in the app. The link clicks are being tracked at hm-dashboard

Comment (marvinmarnold) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: How exactly is hm-dashboard tracking these inbound clicks? UTM? Segment?

Comment (Andrewsopoku) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: When the user go to the HM dashboard screen, it get's logged to segment. I am not sure the UTM situation is applicable here. The users are still within the app although accessing the dashboard.  I have added the UTM tracking though

Comment (marvinmarnold) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: We don't just want to track number of visits to the dashboard, but we need to be able to differentiate which of those are coming from the app. More specifically, we want to know how many registrations and license purchases come following a redirection from the app. Can you figure out the best way for us to measure that? Let me know if you need additional access.

Comment (Andrewsopoku) [reply src/features/hmDashboard/HmDashboardScreen.tsx:17]: 1. Don't you think a different issue needs to be drafted for this?
2. The issue i see is the app track users based on their wallet id whiles the dashboard uses their user id. So we need to agree on a common identifier to be able to create a linkage between the 2.

**Review by marvinmarnold — APPROVED** [review]
Please create a follow-up ticket for whatever you think needs to be done.

**Pull request merged**